apps/aehttp/test/aehttp_dryrun_SUITE.erl

```diff
@@ -114,7 +114,9 @@ spend_txs(Config) ->
     Tx1 = create_spend_tx(APub, EPub, 100000 * aec_test_utils:min_gas_price(), 20000 * aec_test_utils:min_gas_price(), 1, 100),
     Tx2 = create_spend_tx(EPub, APub, 100, 20000 * aec_test_utils:min_gas_price(), 1, 100),
 
-    {ok, 200, #{ <<"results">> := [#{ <<"result">> := <<"ok">> }, #{ <<"result">> := <<"ok">> }] }} =
+    {ok, 200, #{ <<"results">> := [#{ <<"result">> := <<"ok">>,
+                                      <<"type">> := <<"spend">> },
+                                   #{ <<"result">> := <<"ok">> }] }} =
```

Comment:
Does the second map not also contain `<<"type">> := <<"spend">>`?

Reply:
Yes. I tested only on the first one.